### PR TITLE
typescript-timers - Lindsay Jin

### PR DESCRIPTION
**Pull request by lindsay-jin**
Kapture 2024-02-26 at 12 10 43
Kapture 2024-02-26 at 12 08 12

### CODE REVIEW

File: typescript-timers/interval.ts [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  if (!$countdown) throw new Error('The $h1 query has failed.');
+  if ($countdown.textContent === '4') {
+    $countdown.textContent = (count--).toString();
+  } else if ($countdown.textContent === '3') {
+    $countdown.textContent = (count--).toString();
+  } else if ($countdown.textContent === '2') {
+    $countdown.textContent = (count--).toString();
+  } else if ($countdown.textContent === '1')
+    $countdown.textContent = '~Earth Beeeelooowww Us~';
+}
```
Comment (shawnkost) [inline typescript-timers/interval.ts]: Need to call `clearInterval` inside of here

File: typescript-timers/timeout.ts [hunk 2]
```diff
@@ -0,0 +1,8 @@
+const $h1 = document.querySelector('h1');
+if (!$h1) throw new Error('The $h1 query failed.');
+
+$h1.textContent = 'Hello There';
```
Comment (shawnkost) [inline typescript-timers/timeout.ts]: Delete this, we only want this to run inside our `setTimeout`

**Review by shawnkost — CHANGES_REQUESTED** [review]
Looking good Lindsay! Just a couple tweaks here

**Comment by lindsay-jin** [comment]
Edits updated!

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**